playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -44,6 +44,23 @@
   roles:
   - openshift_master_facts
 
+- name: configure vsphere svc account
```

Comment:
It's complaining because you've defined a task outside of the scope of a play in a playbook. Assuming this should only be run on the first master
```
- name: configure vsphere svc account
  hosts: oo_first_master
  tasks:
  - include_role:
      name: openshift_cloud_provider
      tasks_from: vsphere-svc
    when:
    - openshift_cloudprovider_kind in 'vsphere'
    - openshift_version | version_compare('3.9', '>=')
```